documentation/source/user_section/installation.rst

```diff
@@ -24,21 +22,18 @@ Requirements
   * RedHat/CentOS >= 7
   * Windows, see `Install on Windows 10 with WSL`_.
 
-* You need to have ``gcc`` installed. On macOS, we recommend installing `Homebrew <https://brew.sh/>`_ and then run
-  ``brew install gcc``. On Linux, we recommend installing it via your package manager. For example on Debian/Ubuntu:
-  ``apt install gcc``, and on CentOS/RedHat: ``yum -y install gcc``.
+* You need to have ``gcc`` installed.
+    On macOS, we recommend installing `Homebrew <https://brew.sh/>`_ and then run ``brew install gcc``.
```

Comment:
nice change-- did you check if the list rendering on RDT works

Reply:
Normally, this would be in "Details" link in the `docs/readthedocs.org:spinalcordtoolbox` CI check below. But, the check failed, so the warnings will have to be addressed first before this can be rendered. See: https://github.com/neuropoly/spinalcordtoolbox/pull/3197#discussion_r563969893

Reply:
@jcohenadad Good catch.

![Screenshot from 2021-01-27 14-20-45](https://user-images.githubusercontent.com/16181459/106042311-e2295b00-60aa-11eb-8da1-8f97a80e59ee.png)

Reply:
Oops! I didn't mean to turn this into a sub-list, the original was all rendered into the same paragraph for some reason, so I didn't foresee that this could render differently by simply having the line break happen elsewhere on the line. Changed it now to have it all on one line (and re-pushed the last commit), which then renders exactly like the existing single-paragraph output.

Reply:
Ahh, thanks for the clarification. I misunderstood about the sublist. :stuck_out_tongue_closed_eyes: 

It still [looks the same](https://spinalcordtoolbox--3197.org.readthedocs.build/en/3197/user_section/installation.html#requirements) to me, but it's not such a big deal, I think?

Reply:
False alarm. Ctrl + Shift + R is my friend. :facepalm: